src/main/java/lottery/Colors.java

```diff
@@ -0,0 +1,10 @@
+package lottery;
+
+public enum Colors {
+    Red,
+    Green,

```

Comment:
```suggestion
    RED,
    GREEN,
```
enum values are constants and should have proper naming